src/mgos_ili9341_font.c

```diff
@@ -21,104 +21,104 @@ static enum GFXfont_t s_font_type = GFXFONT_NONE;
 
 static bool ili9341_analyzeFont(GFXfont *f) {
   int chars = f->last - f->first;
-  int maxHeight=0, minyo=0, maxWidth=0, maxAdvance=0, minxo=0, maxxo=0, maxhyo=0, minhyo=0, maxyo=0, maxwxo=0, minwxo=0;
-
-  for (int i=0; i<chars; i++) {
-    GFXglyph *glyph  = f->glyph+i;
-    uint8_t  w  = glyph->width;
-    uint8_t  h  = glyph->height;
-    int8_t   xo = glyph->xOffset;
-    int8_t   xa = glyph->xAdvance;
-    int8_t   yo = glyph->yOffset;
-
-    LOG(LL_DEBUG, ("char=0x%02x '%c' w=%d h=%d xOffset=%d yOffset=%d xAdvance=%d", i+f->first, i+f->first, w, h, xo, yo, xa));
-    if (h+yo>maxhyo) maxhyo=h+yo;
-    if (-(h+yo)>minhyo) minhyo=h+yo;
-    if (w+xo>maxwxo) maxwxo=w+xo;
-    if (-(w+xo)>minwxo) minwxo=w+xo;
-
-    if (yo<minyo) minyo=yo;
-    if (yo>maxyo) maxyo=yo;
-    if (xo<minxo) minxo=xo;
-    if (xo>maxxo) maxxo=xo;
-    if (xo>maxxo) maxxo=xo;
-    if (h>maxHeight) maxHeight=h;
-    if (w>maxWidth) maxWidth=w;
-    if (xa>maxAdvance) maxAdvance=xa;
+  int maxHeight = 0, minyo = 0, maxWidth = 0, maxAdvance = 0, minxo = 0, maxxo = 0, maxhyo = 0, minhyo = 0, maxyo = 0, maxwxo = 0, minwxo = 0;
+
+  for (int i = 0; i < chars; i++) {
+    GFXglyph *glyph = f->glyph + i;
+    uint8_t w = glyph->width;
+    uint8_t h = glyph->height;
+    int8_t xo = glyph->xOffset;
+    int8_t xa = glyph->xAdvance;
+    int8_t yo = glyph->yOffset;
+
+    LOG(LL_DEBUG, ("char=0x%02x '%c' w=%d h=%d xOffset=%d yOffset=%d xAdvance=%d", i + f->first, i + f->first, w, h, xo, yo, xa));
+    if (h + yo > maxhyo) maxhyo = h + yo;
+    if (-(h + yo) > minhyo) minhyo = h + yo;
+    if (w + xo > maxwxo) maxwxo = w + xo;
+    if (-(w + xo) > minwxo) minwxo = w + xo;
+
+    if (yo < minyo) minyo = yo;
+    if (yo > maxyo) maxyo = yo;
+    if (xo < minxo) minxo = xo;
+    if (xo > maxxo) maxxo = xo;
+    if (xo > maxxo) maxxo = xo;
+    if (h > maxHeight) maxHeight = h;
+    if (w > maxWidth) maxWidth = w;
+    if (xa > maxAdvance) maxAdvance = xa;
   }
   LOG(LL_DEBUG, ("maxHeight=%d maxWidth=%d maxAdvance=%d minxo=%d maxxo=%d minyo=%d maxyo=%d maxhyo=%d, minhyo=%d", maxHeight, maxWidth, maxAdvance, minxo, maxxo, minyo, maxyo, maxhyo, minhyo));
-  f->font_height = maxhyo-minyo;
-  f->font_width = maxAdvance>maxWidth?maxAdvance:maxWidth;
+  f->font_height = maxhyo - minyo;
+  f->font_width = maxAdvance > maxWidth ? maxAdvance : maxWidth;
   f->font_min_xOffset = minxo;
   f->font_min_yOffset = minyo;
-  LOG(LL_INFO, ("Fontsize: HxW = %dx%d; min_xOffset=%d, min_yOffset=%d, there are %d chars in this font", f->font_height,f->font_width, f->font_min_xOffset, f->font_min_yOffset, chars));
+  LOG(LL_INFO, ("Fontsize: HxW = %dx%d; min_xOffset=%d, min_yOffset=%d, there are %d chars in this font", f->font_height, f->font_width, f->font_min_xOffset, f->font_min_yOffset, chars));
 
```

Comment:
It is very difficult to review the code if you are mixing readability refactoring in. Can you scrub this PR so that all non-code-changes (like these) are not diffs?

Reply:
Sorry about that :(